src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.spec.ts

```diff
@@ -57,32 +61,48 @@ describe('NewScannerFormComponent', () => {
     let el = fixture.nativeElement.querySelector('clr-control-error');
     expect(el).toBeTruthy();
   });
-
-  it('name should be valid', () => {
+  it('name should be existed', fakeAsync(() => {
+    let nameInput = fixture.nativeElement.querySelector('#scanner-name');
+    nameInput.value = "test1";
+    nameInput.dispatchEvent(new Event('input'));
+    nameInput.blur();
+    nameInput.dispatchEvent(new Event('blur'));
+    let el = null;
+    setTimeout(() => {
+      el = fixture.nativeElement.querySelector('#name-error');
+      expect(el).toBeTruthy();
+    }, 20000);
+    tick(20000);
```

Comment:
Suggest use async done to expect the value

Reply:
"async done" is not suitable for this case.